packages/graphback/src/schema/schemaTemplate.ts

```diff
@@ -30,53 +30,107 @@ const filters = (defs: TargetType[]): string => {
 
 const imports = `import gql from 'graphql-tag'`
 
+/**
+ * Patch together generated and custom queries
+ * @param queries queries generated from the types (CRUD)
+ * @param custom custom queries inputted by the user
+ */
+const generateQueries = (queries: string[], custom: Field[]) => {
+  if(custom.length) {
+    return `type Query {
+    ${queries.join('\n    ')}
+    ## Custom queries
+    ${custom.map((f: Field) => {
+      return maybeNullFieldArgs(f)
+    }).join('\n    ')}
+  }`
+  } else {
+    return `type Query {
+    ${queries.join('\n    ')}
+  }`
+  }
+}
+
+/**
+ * Patch together generated and custom mutations
+ * @param mutations mutations generated from the types (CRUD)
+ * @param custom custom mutations inputted by the user
+ */
+const generateMutations = (mutations: string[], custom: Field[]) => {
+  if(custom.length) {
+    return `type Mutation {
+    ${mutations.join('\n    ')}
+    ## Custom mutations
+    ${custom.map((f: Field) => {
```

Comment:
I think this will be hard to maintain. Let's do all the required preprocessing before.

Reply:
I think if else is required, but doing the preprocessing